docs/gettingstarted.rst

```diff
@@ -0,0 +1,91 @@
+Getting started with ilastik API and OMERO
+==========================================
+
+**Description**
+---------------
+
+We will use a Python script showing how to analyze data stored in an OMERO server
+using the ilastik API. The code snippets are extracted from the Python script
+:download:`pixels_classification.py <../scripts/pixels_classification.py>`. A notebook
+is also available, see `pixels_classification.ipynb <https://mybinder.org/v2/gh/ome/omero-guide-ilastik/master?filepath=notebooks>`_.
```

Comment:
Also rename the notebook accordingly https://github.com/ome/omero-guide-ilastik/pull/10/files#r413807260 ?

Reply:
done